### PR TITLE
Manual promo blocks now respect focal point

### PR DESCRIPTION
**Pull request by annasherman**
**If you have not filled out the checklist below, the pr is not ready for review.**

Manual promo blocks now respect focal point

_Information about what you changed for this PR_

## Jira Ticket

- [THEMES-896](https://arcpublishing.atlassian.net/browse/THEMES-896)

## Acceptance Criteria

GIVEN I am an Editor, WHEN I add a [focal point](https://docs.arcxp.com/alc?sys_kb_id=83344050476ad510a87626c2846d4310&id=kb_article_view&sysparm_rank=3&sysparm_tsqueryId=c5ef9085476b9d10a87626c2846d43e2) to my image in Photo Center, THEN:

    All manual promo blocks with images respect the focal point on rendering in the v2 architecture

        The focal point can be added on the image component so it will apply to all block images

        Blocks to be tested

            extra-large-manual-block

            large-manual-promo-block

            medium-manual-promo-block

            small-manual-promo-block

    If there is no focal point, then the smart crop is maintained - see

## Test Steps

- Add test steps a reviewer must complete to test this PR

1. Checkout this branch `git checkout themes-896`
2. Run fusion repo with linked blocks `npx fusion start -f -l @wpmedia/small-manual-promo-block,@wpmedia/extra-large-manual-promo-block,@wpmedia/medium-manual-promo-block,@wpmedia/large-manual-promo-block`
3. Add the manual promo blocks to a page. Configure content => there are two images set up with a focal point. You can search for "focal point" to see the camel & the pyramid. Make sure the block respect the focal point that has been set. 

### Before

Nothing happened in manual promo blocks when focal point was set

### After

Focal point is respected.
Screenshot 2024-03-13 at 12 05 22 PM

## Author Checklist

_The author of the PR should fill out the following sections to ensure this PR is ready for review._

- [x] Confirmed all the test steps a reviewer will follow above are working.
- [x] Confirmed there are no linter errors. Please run `npm run lint` to check for errors. Often, `npm run lint:fix` will fix those errors and warnings.
- [x] Ran this code locally and checked that there are not any unintended side effects. For example, that a CSS selector is scoped only to a particular block.
- [x] Confirmed this PR has reasonable code coverage. You can run `npm run test:coverage` to see your progress.
  - [x] Confirmed this PR has unit test files
  - [x] Ran `npm run test`, made sure all tests are passing
  - [ ] If the amount of work to write unit tests for this change are excessive,
        please explain why (so that we can fix it whenever it gets refactored).
- [ ] Confirmed relevant documentation has been updated/added.

## Reviewer Checklist

_The reviewer of the PR should copy-paste this template into the review comments on review._

- [ ] Linting code actions have passed.
- [ ] Ran the code locally based on the test instructions.
  - [ ] I don’t think this is needed to be tested locally. For example, a padding style change (storybook?) or a logic change (write a test).
- [ ] I am a member of the engine theme team so that I can approve and merge this. If you're not on the team, you won't have access to approve and merge this pr.
- [ ] Looked to see that the new or changed code has code coverage, specifically. We want the global code coverage to keep on going up with targeted testing.

[THEMES-896]: https://arcpublishing.atlassian.net/browse/THEMES-896?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

**Review by blakeganderson — DISMISSED** [review]
LGTM, but there are several warnings from Github actions. Not sure if those need to be addressed before merging

**Comment by vgalatro** [comment]
If `Test Blocks And Linting` is failing, the block won't publish when this is merged in @blakeganderson / @annasherman . It looks like it's just a linting issue, so should be a simple fix.

**Review by vgalatro — CHANGES_REQUESTED** [review]
Check the github action for the linting errors. You can also run `npm run lint:changed-feature-branch` locally to see all the issues.

File: blocks/small-manual-promo-block/features/small-manual-promo/default.jsx [hunk 1]
```diff
@@ -135,13 +141,13 @@ const SmallManualPromo = ({ customFields }) => {
 				<Grid as="article" className={containerClassNames}>
 					{["below", "right"].includes(imagePosition) ? (
 						<>
-							<PromoHeading />
-							<PromoImage />
+							<PromoHeading showHeadline={showHeadline} headline={headline} linkURL={linkURL} newTab={newTab} registerSuccessEvent={registerSuccessEvent} />
+							<PromoImage showImage={showImage} searchableField={searchableField} imageParams={imageParams} linkURL={linkURL} newTab={newTab} registerSuccessEvent={registerSuccessEvent} showHeadline={showHeadline} />
```
Comment (vgalatro) [inline blocks/small-manual-promo-block/features/small-manual-promo/default.jsx]: These props are a bit hard to read, could you put them on their own lines like so?

```suggestion
							<PromoHeading
								showHeadline={showHeadline}
								headline={headline}
								linkURL={linkURL}
								newTab={newTab}
								registerSuccessEvent={registerSuccessEvent}
							/>
							<PromoImage
								showImage={showImage}
								searchableField={searchableField}
								imageParams={imageParams}
								linkURL={linkURL}
								newTab={newTab}
								registerSuccessEvent={registerSuccessEvent}
								showHeadline={showHeadline}
							/>
```

File: blocks/small-manual-promo-block/features/small-manual-promo/default.jsx [hunk 2]
```diff
@@ -135,13 +141,13 @@ const SmallManualPromo = ({ customFields }) => {
...
 						</>
 					) : (
 						<>
-							<PromoImage />
-							<PromoHeading />
+							<PromoImage showImage={showImage} searchableField={searchableField} imageParams={imageParams} linkURL={linkURL} newTab={newTab} registerSuccessEvent={registerSuccessEvent} showHeadline={showHeadline} />
```
Comment (vgalatro) [inline blocks/small-manual-promo-block/features/small-manual-promo/default.jsx]: Same suggestion as above.

**Review by vgalatro — CHANGES_REQUESTED** [review]
@annasherman , just a minor formatting change. Everything else looks and works great!

File: blocks/small-manual-promo-block/features/small-manual-promo/default.jsx (hunk 2 above)
Comment (annasherman) [reply blocks/small-manual-promo-block/features/small-manual-promo/default.jsx]: fixed!

**Review by vgalatro — APPROVED** [review]
LGTM!

**Pull request merged**